label_sync/labels.yaml

```diff
@@ -535,6 +535,24 @@ labels:
     target: prs
     prowPlugin: size
     addedBy: prow
+  - color: ee5500
```

Comment:
I am not sure about these colors. The existing stage labels can be found at: https://github.com/kubernetes/features/labels?page=3&sort=name-asc.

In this PR, currently:

- stage/alpha has the same color as size/XL
- stage/beta has the same color as size/M
- stage/stable has the same color as size/XS